app/templates/profile.html

```diff
@@ -219,6 +219,15 @@ <h6 class="card-title text-center mb-1">{{ user.full_name }}</h6>
               <li class="list-group-item list-group-item-action no-border">
                 <a class="text-decoration-none text-secondary" href="#">Export my calendar</a>
               </li>
+              {% if not user.is_active_exercise %}
+              <li class="list-group-item list-group-item-action no-border">
+                <a class="text-decoration-none text-secondary" href="./exercise/start">Start Exercise</a>
```

Comment:
Prefer `url_for`

Reply:
No, If I write url_for , it did not find the route and throw an exception